packages/table/components/CheckboxTable.tsx

```diff
@@ -62,11 +68,14 @@ class CheckboxTable extends React.PureComponent<CheckboxTableProps> {
   private getHeaderCheckbox() {
     const selectedRows = this.props.selectedRows || [];
     let isSelected = false;
```

Comment:
I know I'm late to the party 😅 could this `isSelected` be a part of `state` instead of being local mutable variable?

Reply:
Definitely. I was stubbornly trying to avoid state